python/paddle/fluid/metrics.py

```diff
@@ -116,7 +116,7 @@ def __init__(self, name=None, **kwargs):
         super(CompositeMetric, self).__init__(name, kwargs)
         self._metrics = []
 
-    def add_metric(self, metric):
+    def update(self, metric):
```

Comment:
I add the interface `add_metric` on purpose, means that add a new one metric to Composed metrics.

Reply:
reverted